--- lang/c++/CMakeLists_origin.txt	2020-04-27 14:36:49.239533672 +0900
+++ lang/c++/CMakeLists.txt	2020-04-27 14:31:12.316056718 +0900
@@ -124,61 +124,61 @@
 
 target_link_libraries (precompile avrocpp_s ${Boost_LIBRARIES} ${SNAPPY_LIBRARIES})
 
-macro (gen file ns)
-    add_custom_command (OUTPUT ${file}.hh
-        COMMAND avrogencpp
-            -p -
-            -i ${CMAKE_CURRENT_SOURCE_DIR}/jsonschemas/${file}
-            -o ${file}.hh -n ${ns} -U
-        DEPENDS avrogencpp ${CMAKE_CURRENT_SOURCE_DIR}/jsonschemas/${file})
-    add_custom_target (${file}_hh DEPENDS ${file}.hh)
-endmacro (gen)
-
-gen (empty_record empty)
-gen (bigrecord testgen)
-gen (bigrecord_r testgen_r)
-gen (bigrecord2 testgen2)
-gen (tweet testgen3)
-gen (union_array_union uau)
-gen (union_map_union umu)
-gen (union_conflict uc)
-gen (recursive rec)
-gen (reuse ru)
-gen (circulardep cd)
-gen (tree1 tr1)
-gen (tree2 tr2)
-gen (crossref cr)
-gen (primitivetypes pt)
+#macro (gen file ns)
+#    add_custom_command (OUTPUT ${file}.hh
+#        COMMAND avrogencpp
+#            -p -
+#            -i ${CMAKE_CURRENT_SOURCE_DIR}/jsonschemas/${file}
+#            -o ${file}.hh -n ${ns} -U
+#        DEPENDS avrogencpp ${CMAKE_CURRENT_SOURCE_DIR}/jsonschemas/${file})
+#    add_custom_target (${file}_hh DEPENDS ${file}.hh)
+#endmacro (gen)
+#
+#gen (empty_record empty)
+#gen (bigrecord testgen)
+#gen (bigrecord_r testgen_r)
+#gen (bigrecord2 testgen2)
+#gen (tweet testgen3)
+#gen (union_array_union uau)
+#gen (union_map_union umu)
+#gen (union_conflict uc)
+#gen (recursive rec)
+#gen (reuse ru)
+#gen (circulardep cd)
+#gen (tree1 tr1)
+#gen (tree2 tr2)
+#gen (crossref cr)
+#gen (primitivetypes pt)
 
 add_executable (avrogencpp impl/avrogencpp.cc)
 target_link_libraries (avrogencpp avrocpp_s ${Boost_LIBRARIES} ${SNAPPY_LIBRARIES})
 
-enable_testing()
-
-macro (unittest name)
-    add_executable (${name} test/${name}.cc)
-    target_link_libraries (${name} avrocpp ${Boost_LIBRARIES} ${SNAPPY_LIBRARIES})
-    add_test (NAME ${name} WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-        COMMAND ${CMAKE_CURRENT_BINARY_DIR}/${name})
-endmacro (unittest)
-
-unittest (buffertest)
-unittest (unittest)
-unittest (SchemaTests)
-unittest (LargeSchemaTests)
-unittest (CodecTests)
-unittest (StreamTests)
-unittest (SpecificTests)
-unittest (DataFileTests)
-unittest (JsonTests)
-unittest (AvrogencppTests)
-unittest (CompilerTests)
-
-add_dependencies (AvrogencppTests bigrecord_hh bigrecord_r_hh bigrecord2_hh
-    tweet_hh
-    union_array_union_hh union_map_union_hh union_conflict_hh
-    recursive_hh reuse_hh circulardep_hh tree1_hh tree2_hh crossref_hh
-    primitivetypes_hh empty_record_hh)
+#enable_testing()
+#
+#macro (unittest name)
+#    add_executable (${name} test/${name}.cc)
+#    target_link_libraries (${name} avrocpp ${Boost_LIBRARIES} ${SNAPPY_LIBRARIES})
+#    add_test (NAME ${name} WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+#        COMMAND ${CMAKE_CURRENT_BINARY_DIR}/${name})
+#endmacro (unittest)
+#
+#unittest (buffertest)
+#unittest (unittest)
+#unittest (SchemaTests)
+#unittest (LargeSchemaTests)
+#unittest (CodecTests)
+#unittest (StreamTests)
+#unittest (SpecificTests)
+#unittest (DataFileTests)
+#unittest (JsonTests)
+#unittest (AvrogencppTests)
+#unittest (CompilerTests)
+#
+#add_dependencies (AvrogencppTests bigrecord_hh bigrecord_r_hh bigrecord2_hh
+#    tweet_hh
+#    union_array_union_hh union_map_union_hh union_conflict_hh
+#    recursive_hh reuse_hh circulardep_hh tree1_hh tree2_hh crossref_hh
+#    primitivetypes_hh empty_record_hh)
 
 include (InstallRequiredSystemLibraries)
 
